Description: Let TestProbitCG test run for a bit longer, fixing FTBFS
Author: Michael Terry <michael.terry@canonical.com>
Bug: https://github.com/statsmodels/statsmodels/issues/109
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=648845

Index: statsmodels-0.3.1/scikits/statsmodels/discrete/tests/test_discrete.py
===================================================================
--- statsmodels-0.3.1.orig/scikits/statsmodels/discrete/tests/test_discrete.py	2011-11-15 09:57:09.346314408 -0500
+++ statsmodels-0.3.1/scikits/statsmodels/discrete/tests/test_discrete.py	2011-11-15 10:01:45.998317336 -0500
@@ -227,7 +227,7 @@
         res2.probit()
         cls.res2 = res2
         cls.res1 = Probit(data.endog, data.exog).fit(method="cg",
-            disp=0, maxiter=250)
+            disp=0, maxiter=500)
 
 class TestProbitNCG(CheckModelResults):
     @classmethod
